automated/linux/fdisk/fdisk.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+SKIP_INSTALL="false"
+
+DEVICE=""
+FILESYSTEM="ext4"
+SIZES=""
+
+usage() {
+    echo "Usage: $0 [-d </dev/sdb>]
+                    [-f <ext4>]
+                    [-s <true|false>]
+                    [-z <+10G +5G>]" 1>&2
+    exit 1
+}
+
+# create_partitions "/dev/sdb" "+5G +5G"
+create_partitions() {
+    echo
+    echo "Creating partition: ${DEVICE}"
+    skip_list="format-partitions"
+    partition_disk "${DEVICE}" "${SIZES}"
```

Comment:
please check if SIZES is defined and not empty.

Reply:
if SIZES="" then single partition with total available size.